config/formats.js

```diff
@@ -689,7 +689,8 @@ let Formats = [
 			set.species = template.species;
 			return problems;
 		},
-		onModifyTemplate(template, target, effect) {
+		onModifyTemplate(template, target, source, effect) {
+			if (!effect || !target) return;
```

Comment:
I'm pretty sure `!effect` is unnecessary.

Also how does Cross Evolution work? Should `/dt` just ignore it entirely?

Reply:
i dont really know how it works, its one of the few OMs i dont care enough to look into.

and yeah `!effect` is unnecessary, ill remove it

Reply:
Aha, it works on nicknames which are unavailable in `/dt`. That makes sense.

Reply:
removed `!effect` @Zarel